iptables/iptables_test.go

```diff
@@ -119,7 +123,20 @@ func TestRules(t *testing.T) {
 }
 
 func runRulesTests(t *testing.T, ipt *IPTables) {
-	t.Logf("testing iptables (hasWait=%t, hasCheck=%t)", ipt.hasWait, ipt.hasCheck)
+	t.Logf("testing %s (hasWait=%t, hasCheck=%t)", getIptablesCommand(ipt.proto), ipt.hasWait, ipt.hasCheck)
+
+	var address1, address2, subnet1, subnet2 string
+	if ipt.proto == ProtocolIPv6 {
+		address1 = "2001:db8::1/128"
+		address2 = "2001:db8::2/128"
+		subnet1 = "fd00:a::/48"
+		subnet2 = "fd00:b::/48"
+	} else {
+		address1 = "203.0.113.1/32"
+		address2 = "203.0.113.2/32"
+		subnet1 = "10.1.0.0/16"
```

Comment:
You mentioned changing to RFC5737, any reasons for keeping these (and the v6 above) from RFC1918 instead? Is iptables unhappy with that?